include/boost/json/serializer.hpp

```diff
@@ -228,10 +238,90 @@ class serializer
 
         @param size The maximum number of characters
         to write to the memory pointed to by `dest`.
+
+        @param opts The options for serialization.
     */
     BOOST_JSON_DECL
     string_view
-    read(char* dest, std::size_t size);
+    read(char* dest, std::size_t size, const serialize_options & opts = {});
```

Comment:
The options should be for the whole serializer not for each individual call to `read`